tests/test_async_client.py

```diff
@@ -8,12 +8,19 @@
 pytestmark = [pytest.mark.anyio, pytest.mark.network]
 
 
-@pytest.mark.parametrize("qos", [QoS.AT_MOST_ONCE, QoS.AT_LEAST_ONCE, QoS.EXACTLY_ONCE])
-async def test_publish_subscribe(qos: QoS) -> None:
+@pytest.mark.parametrize(
+    "qos_sub", [QoS.AT_MOST_ONCE, QoS.AT_LEAST_ONCE, QoS.EXACTLY_ONCE]
+)
+@pytest.mark.parametrize(
+    "qos_pub", [QoS.AT_MOST_ONCE, QoS.AT_LEAST_ONCE, QoS.EXACTLY_ONCE]
+)
+async def test_publish_subscribe(qos_sub: QoS, qos_pub: QoS) -> None:
     async with AsyncMQTTClient() as client:
-        async with client.subscribe("test/+") as messages:
-            await client.publish("test/text", "test åäö", qos=qos)
-            await client.publish("test/binary", b"\x00\xff\x00\x1f", qos=qos)
+        if qos_pub > client.maximum_qos:
+            return  # TODO add pytest.skip
```

Comment:
```suggestion
            return  # TODO add pytest.skip

```

Reply:
Would you mind allowing me to make changes too so I can make these trivial edits without wasting your time on them?

Reply:
Well yes sure, the problem is that github doesn't even ask me whether to allow that. I haven't yet figured out where *that* setting has moved to these days, it used to be right on the create-a-PR page.

Reply:
Is [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) helpful?

Reply:
changed

Reply:
> Is [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) helpful?

This option isn't even displayed when I look at the PR(s).

Seems like I'll have to ask github support.

Reply:
Ah. The reason was that I submitted them from my MoaT org collection instead of my personal one. *Sigh*.

No explanation why that should be so …